### PR TITLE
refactor: Set appium as peer dependency

### PR DESCRIPTION
**Pull request by mykola-mokhnach**

### CODE REVIEW

File: test/server.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { main as startAppium } from 'appium';
+
+async function startServer (port, address, relaxedSecurityEnabled = false) {
+  // https://github.com/appium/appium/blob/2.0/packages/schema/lib/appium-config.schema.json
+  return await startAppium({
+    port,
+    address,
+    'relaxed-security': relaxedSecurityEnabled,
```
Comment (jlipps) [inline test/server.js:8]: kinda surprised this takes kebab instead of camel case version of the opts at this point.

**Review by jlipps — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -64,6 +64,8 @@
     "@babel/core": "^7.16.0",
     "@babel/eslint-parser": "^7.16.3",
     "@semantic-release/git": "^10.0.1",
+    "appium": "^2.0.0-beta.40",
+    "appium-mac2-driver": "file:.",
```
Comment (boneskull) [inline package.json:68]: what's this for?

**Review by boneskull — APPROVED** [review]
this looks ok, but I am a little confused on the PR topic and commit msgs

File: test/server.js (hunk 1 above)
Comment (boneskull) [reply test/server.js:8]: yeah, I'm not sure that's right.  I don't recall if it'll work offhand.  The parameter to `startAppium` (which is presumably the `main()` function) is an [`Args`](https://github.com/appium/appium/blob/2.0/packages/appium/types/cli.ts#L137) object, which is a union that includes `ServerArgs`--and all props in `ServerArgs` are camel-cased.

**Review by boneskull — CHANGES_REQUESTED** [review]
I don't think the `relaxed-caps` arg is correct.  please verify

File: test/server.js (hunk 1 above)
Comment (mykola-mokhnach) [reply test/server.js:8]: Got it. Changed.

File: package.json (hunk 2 above)
Comment (mykola-mokhnach) [reply package.json:68]: to be able to start a local server for e2e tests, a hint from @jlipps : https://appium.slack.com/archives/C02956V3H/p1655144800931379

**Review by KazuCocoa — APPROVED** [review]

**Pull request merged**